### ISSUE TITLE
Extending the data to 2020

### ISSUE THREAD

**Issue by yingkaisha** [pangeo-data/WeatherBench#27]
Hi, 
I found the 1.4-degree data is available till the end of 2018, and would like to extend it to 2020. So I can hold more data for training.

How is the original ERA5 re-gridded? Based on _Rasp et al., [2020]_, should one use xESMF with `method=bilinear`?

Thank you.

**Comment by raspstephan** [comment]
Yes, I used bilinear interpolation with xesmf. You can use `src/regrid.py`.

You can use the download and regridding workflow defined in the snakemake file. The repo README has some brief instructions. You would only have to change the `all_years` at the top.
